src/content/atom/index.html

```diff
@@ -15,8 +15,10 @@
 	<h4>Install using Atom</h4>
 	<ol>
 		<li>Go to <code>Atom -> Preferences...</code></li>
-		<li>Then select the <code>Themes</code> tab</li>
-		<li>Enter <code>Dracula</code> in the search box</li>
+		<li>Then select the <code>Install</code> tab</li>
+		<li>Click the <code>Themes</code> button to the right of the search box </li>
+		<li>Enter <code>Dracula-theme</code> in the search box</li>
```

Comment:
This should be `dracula-theme` not `Dracula-theme`. The package itself is all lowercase.